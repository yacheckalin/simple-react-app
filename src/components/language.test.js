import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import Language from "./language";

describe("<Language />", () => {
  test("displays and shows properly", () => {
    const wrapper = shallow(<Language />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
