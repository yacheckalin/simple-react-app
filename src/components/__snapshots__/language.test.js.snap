// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Language /> displays and shows properly 1`] = `
<styled.div>
  <button
    onClick={[Function]}
  >
    EN
  </button>
  |
  <button
    onClick={[Function]}
  >
    RU
  </button>
</styled.div>
`;
